### ISSUE TITLE
Add option to switch workspaces (also backwards) similar to switching windows

### ISSUE THREAD

**Issue by elementaryBot** [elementary/switchboard-plug-keyboard#54]
I'd prefer to have an option to be able to switch between workspaces similar to how one switches between windows. The logical default choice would be cmd+tab which would cycle between used workspaces, while also showing the multitasking overview while cmd is pressed (just like Plank is used as window overview when the alt key is still pressed after alt+tab'ing).

As I (and probably others) rarerly use more than 3 workspaces, cycling wouldn't be an issue, cmd+tab is much easier to reach than cmd+left/right and seems more intuitive.

Launchpad Details: [#LP1485805](https://bugs.launchpad.net/bugs/1485805) quassy - 2015-08-17 23:50:09 +0000

**Comment by elementaryBot** [comment]
You can use cmd+number keys which is faster. Cmd+tab is also possible already but the keystroke is blocked by a Gnome gsettings key. Dunno which right know.

Launchpad Details: [#LPC](https://bugs.launchpad.net/switchboard-plug-keyboard/+bug/1485805/comments/1) Djax - 2015-08-18 09:08:42 +0000

**Comment by elementaryBot** [comment]
cmd+number still requires me to specifically aim for one desktop, especially because there is alt+tab, cmd+tab seems better. cmd+tab is not possible as there is only "Switch to right" which does not cycle when at the end of the workspace list. So this bug is either about making "Switch to right" cycle when at the end of the list, or adding a new option "Switch Workspaces" which does just that (like "Switch Windows").

Launchpad Details: [#LPC](https://bugs.launchpad.net/switchboard-plug-keyboard/+bug/1485805/comments/2) quassy - 2015-08-18 13:19:29 +0000

**Comment by elementaryBot** [comment]
Like i said this "switch workspaces" is already there but is blocked by a Gnome gsettings key. So remove Cmd+tab from the org.gnome.desktop.wm.keybindings.switch-applications key and you can use what you want.

Launchpad Details: [#LPC](https://bugs.launchpad.net/switchboard-plug-keyboard/+bug/1485805/comments/3) Djax - 2015-08-18 17:35:23 +0000

**Comment by elementaryBot** [comment]
Thanks, worked for me. But the report is still valid, to put it differently (and the reason this is against the keyboard plug not Gala): The shortcut should be available in the keyboard plug to customize.

Launchpad Details: [#LPC](https://bugs.launchpad.net/switchboard-plug-keyboard/+bug/1485805/comments/4) quassy - 2015-08-18 17:51:28 +0000

**Comment by elementaryBot** [comment]
Ok, removed the duplicate. I don't think this should be customizable but it's not me to decide that...

Launchpad Details: [#LPC](https://bugs.launchpad.net/switchboard-plug-keyboard/+bug/1485805/comments/5) Djax - 2015-08-18 22:31:40 +0000

**Comment by danrabbit** [comment]
Closing as a duplicate of #16
